Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM node:10.15.3
+
+# Create app directory
+WORKDIR /platform
+
+COPY . .
+
+WORKDIR /platform/projects/ngrx.io	
+
+# Install all the dependencies, boilerplate, stackblitz, zips and run dgeni on the docs.
+RUN yarn setup
+
+# Create a production build of the application (after installing dependencies, boilerplate, etc)
```

Comment:
This step isn't required and is handled by `yarn start:docker`

Reply:
> This step isn't required and is handled by `yarn start:docker`

Thanks. Removed the step from the dockerfile